Gopkg.toml

```diff
@@ -0,0 +1,27 @@
+[[constraint]]
+  name = "github.com/golang-migrate/migrate"
+  version = "3.5.4"
+
+[[constraint]]
+  name = "github.com/golang/protobuf"
+  version = "1.2.0"
+
+[[constraint]]
+  name = "github.com/jmoiron/sqlx"
+  version = "1.2.0"
+
+[[constraint]]
+  name = "github.com/kshvakov/clickhouse"
+  version = "1.3.5"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/percona/go-mysql"
+
+[[constraint]]
+  branch = "master"
+  name = "golang.org/x/net"
+
+[[constraint]]
+  name = "google.golang.org/grpc"
+  version = "1.17.0"
```

Comment:
First, I think we should add prune options to reduce the number of vendored files. For example:
```
[prune]
  go-tests = true
  non-go = true
  unused-packages = true
```
You may need to comment out the last two to keep vendored `.proto` files if you need them.

Second, I think we should remove default constraints. If we use the latest released version or master, there is no need to mention this constraint explicitly.

Reply:
Please remove all `[[constraint]]` from that file.

Reply:
Removed.